docs/website/docs/user-guides/advanced/debugging-with-openshift-toolkit.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Debugging an Application with OpenShift Toolkit
+sidebar_position: 7
+---
+
+Debugging is an unavoidable part of development, and it can prove even more difficult when developing an application that runs remotely.
+
+However, this task is made absurdly simple with the help of OpenShift Toolkit.
+
+## OpenShift Toolkit
+[OpenShift Toolkit](https://github.com/redhat-developer/intellij-openshift-connector) is an IDE plugin that allows you to do all things that `odo` does, i.e. create, test, debug and deploy cloud-native applications on a cloud-native environment in simple steps.
+`odo` enables this plugin to do what it does.
+
+## Prerequisites
+1. [You have logged in to your cluster](../quickstart/nodejs.md#step-1-connect-to-your-cluster-and-create-a-new-namespace-or-project).
+2. [You have initialized a Node.js application with odo](../quickstart/nodejs.md#step-2-initializing-your-application-odo-init).
```

Comment:
IMO, the prerequisite is actually to have an application initialized with `odo`. So I'd be more generic here, and specify Node.js as an example.

```suggestion
2. You have [initialized an application with `odo`](/docs/command-reference/init), for example [the Node.JS quickstart application](../quickstart/nodejs.md#step-2-initializing-your-application-odo-init).
```